api/index.js

```diff
@@ -22,6 +22,7 @@ module.exports = async (req, res) => {
     svgfill,
     desc,
     scale,
+    rating,
```

Comment:
Can we use the existing text instead of rating? What do you think?

Reply:
I think `text` won't be an appropriate term, that is why I have added `rating` as `text` implies some kind of string. We can think another term like `level` ?

Reply:
Yea but can we use existing params since we might not want to introduce newer variables just for one component?

Reply:
done in [4bd3d4f](https://github.com/harish-sethuraman/readme-components/pull/12/commits/4bd3d4f1245cb61fd9bd564deeddf106069f9e4c)